### PR TITLE
[ONNX] Improve error handling for adaptive_pool

### PR DESCRIPTION
**Pull request by neginraoof**
This would also improve error handling for interpolate with 'area' mode.

### CODE REVIEW

**Review by BowenBao — APPROVED** [review]
LG, thanks

**Review by bzinodev — APPROVED** [review]

**Comment by neginraoof** [comment]
@bzinodev
The failure in Jenkins: pr/pytorch-linux-bionic-rocm3.7-py3.6
is unrelated. I see this failure on other recent PR's as well.

Would you please let me know if this currently blocking merge? Or is there anything I can do to unblock this?

**Comment by neginraoof** [comment]
@bzinodev 
Can we please merge this PR?
Thanks!

**Comment by neginraoof** [comment]
@bzinodev 
Can we please merge this PR?
Thanks!

**Comment by neginraoof** [comment]
@bzinodev Can we merge this please? Thanks!

**Comment by neginraoof** [comment]
cc @bzinodev
This PR is rebased and ready for merge. Thanks a lot.

File: torch/onnx/symbolic_opset9.py
Comment (bzinodev) [inline torch/onnx/symbolic_opset9.py:854]: Old behavior a warning is emitted.  It is changed to a runtime error?  there is one internal test that fails.  Is this something we want?

Comment (bzinodev) [reply torch/onnx/symbolic_opset9.py:854]: Note other ops  all uses _unimplemented

Comment (neginraoof) [reply torch/onnx/symbolic_opset9.py:854]: Before my update, this line in the symbolic conversion was throwing a warning instead of an error. Hence, the ATen node in the graph was not converted to ONNX, and was left as ATen.
By default, this behavior is not accepted. But exporter has a configuration mode that accepts combination of ONNX and ATen nodes. It's called ONNX_ATEN_FALLBACK.
It could be that the internal test case you see is failing is exporting with the ONNX_ATEN_FALLBACK mode.
I'll send a fix.

**Comment by neginraoof** [comment]
@bzinodev Could you please merge the last commit here? There was a typo that was fixed after I addressed your comment.

**Comment by dzhulgakov** [comment]
Hey, this PR unfortunately broke master (probably the missing typo you mentioned). I'm unlanding it, please resubmit the new PR with the fix, @bzinodev or myself can help you land it

**Comment by neginraoof** [comment]
@bzinodev @dzhulgakov The new duplicate PR is: https://github.com/pytorch/pytorch/pull/45874
Could you please help merge this new one?
Thanks!

**Pull request closed without merging**
